bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -87,6 +87,19 @@ Type of CO2 feedback.
 </entry>
 
 
+<entry id="o3_ppbv" type="real" category="clm_physics"
```

Comment:
@sunnivin and I had a chat regarding this. Referring to what we discussed in our zoom call a while ago, all references to o3_ppbv and o3_type should be removed from this PR. It was probably unintentionally that they ended up in this PR. Our plan is to include these in another PR to separate physics implementation from pure technical handy work.

Reply:
Actually, I left this one because I was not 100% sure about what we wanted to do. I think @billsacks told us that somebody is/there are plans for working on the atmospheric forcing, but I think @ziu1986 is interested in having the o3_ppbv option for her scientific explorations. 

Anyway, I totally agree that it should not be included in this PR.